src/styles/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  font-family: $base-font-family;
+  color: $primary-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+
```

it's better to use transition property for element instead of using it directly on hover pseudo class
```suggestion
  transition: transform 300ms;
```